spec/models/task_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
+  let(:title) { "Short text" }
+  let(:description) { "Long text" }
+  let(:due_date) { "11-11-1999" } 
+  
+  context "when valid params" do
+    it "saves task with all params and without done status" do
+      task = Task.new(title: title, description: description, due_date: due_date)
+      expect(task.save).to eq(true)
+    end
+
+    it "saves task with all params and true done status" do
```

Comment:
ok, I only want to check one more situation when user enter true status. Delete them?